README.md

```diff
@@ -134,6 +134,7 @@ This provider exposes quite a few provider-specific configuration options:
 * `subnet_id` - The subnet to boot the instance into, for VPC.
 * `associate_public_ip` - If true, will associate a public IP address to an instance in a VPC.
 * `tags` - A hash of tags to set on the machine.
+* `package_tags` - A hash of tags to set on the ami generated during the package operation.
```

Comment:
Yes you're right it's an naming layer that exists only in the code. Because `vagrant package` has inherently no notion of AMI while AWS has no notion of `package`.

Now, of the two options available for the name ("AMI_tags" and "package_tags"), "AMI" tends to be less precise as it could refer to either the AMI to start the instance or the AMI generated during the `package` operation. Granted that with a bit of context and knowledge of Vagrant the risk of mixing up those two in minimal.
Nevertheless, given that there are two artefacts that are "AMIs", in order to make sure that there is no confusion and that the tags we're creating are indeed for the result of the `package` operation, naming then "package_tags" sounds like the best option.